recipes/configs/mistral/7B_full.yaml

```diff
@@ -25,16 +25,16 @@ checkpointer:
     pytorch_model-00002-of-00002.bin
   ]
   recipe_checkpoint: null
-  output_dir: /tmp/Mistral-7B-v0.1
-  model_type: LLAMA2
+  output_dir: /tmp/Mistral-7B-v0.1/
+  model_type: MISTRAL
```

Comment:
This technically varies from the HF way of defining model type b/c mistral is llama-based. What does model_type=Mistral give us?

Reply:
Less confusion? Defining mistral model to be llama type is a bit confusing I think

Reply:
But it's technically true that it is a llama-type model and this is the accepted standard in HF.

Reply:
I'd definitely dont want to carry over any confusions from there

Reply:
Personally I like the separate model type. Yes it's an identical architecture, but just helps to be explicit. Main q is: is this used as a hard constraint in places? E.g. is it gonna prevent me from loading Mistral weights into a LLAMA2 model type